Gems/Atom/Feature/Common/Assets/Shaders/Materials/EnhancedPBR/EnhancedPBR_Lighting.azsli

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// This #define magic lets you use the EvaluateLighting function in this file without making it the final EvaluateLighting
+// used in your shader. Simply #define EvaluateLighting to your custom definition before including this file
+//
+#ifndef EvaluateLighting
+#define EvaluateLighting(surface, screenPosition)       EvaluateLighting_StandardPBR(surface, screenPosition)
+#endif
+
+#include "../BasePBR/BasePBR_Lighting.azsli"
+
+void InitializeLightingData_StandardPBR(Surface surface, float4 screenPosition, inout LightingData lightingData)
+{
+    // --- Base PBR ---
+    InitializeLightingData_BasePBR(surface, screenPosition, lightingData);
+
+    // --- Standard PBR ---
+
+    lightingData.emissiveLighting = surface.emissiveLighting;
+    lightingData.diffuseAmbientOcclusion = surface.diffuseAmbientOcclusion;
+    lightingData.specularOcclusion = surface.specularOcclusion;
+
+    if(o_clearCoat_feature_enabled)
+    {
+        // Clear coat layer has fixed IOR = 1.5 and transparent => F0 = (1.5 - 1)^2 / (1.5 + 1)^2 = 0.04
+        lightingData.diffuseResponse *= 1.0 - (FresnelSchlickWithRoughness(lightingData.NdotV, float3(0.04, 0.04, 0.04), surface.clearCoat.roughness) * surface.clearCoat.factor);
+    }
+}
+
+void HandleOpacity(Surface surface, inout LightingData lightingData)
+{
+    if (o_opacity_mode == OpacityMode::Blended || o_opacity_mode == OpacityMode::TintedTransparent)
+    {
+        lightingData.diffuseLighting.rgb *= surface.alpha; // pre-multiply diffuse
+        
+        // Add specular. m_opacityAffectsSpecularFactor controls how much the alpha masks out specular contribution.
+        float3 specular = lightingData.specularLighting.rgb;
+        specular = lerp(specular, specular * surface.alpha, surface.opacityAffectsSpecularFactor);
+        lightingData.diffuseLighting.rgb += specular;
+
+        // Increase opacity at grazing angles for surfaces with a low m_opacityAffectsSpecularFactor.
+        // For m_opacityAffectsSpecularFactor values close to 0, that indicates a transparent surface
+        // like glass, so it becomes less transparent at grazing angles. For m_opacityAffectsSpecularFactor
+        // values close to 1.0, that indicates the absence of a surface entirely, so this effect should
+        // not apply.
+        float fresnelAlpha = FresnelSchlickWithRoughness(lightingData.NdotV, surface.alpha, surface.roughnessLinear).x;
+        float alpha = lerp(fresnelAlpha, surface.alpha, surface.opacityAffectsSpecularFactor);
+
+        if (o_opacity_mode == OpacityMode::Blended)
+        {
+            // [GFX_TODO ATOM-13187] PbrLighting shouldn't be writing directly to render targets. It's confusing when
```

Comment:
I think this comment can be removed since now this is being written to the surface struct unless I'm misunderstanding the comment

Reply:
Will fix